modules/android/partials/release-notes/couchbase-mobile-android-release-note.3.1.8.adoc

```diff
@@ -0,0 +1,21 @@
+[#maint-3-1-8]
+== 3.1.8 -- May 2024
+
+Version 3.1.8 for {param-title} delivers the following features and enhancements:
+
+NOTE: For an overview of the latest features offered in Couchbase Lite 3.1, see xref:ROOT:cbl-whatsnew.adoc[New in 3.1]
+
+
+=== Enhancements
+
+None for this release
+
+=== Issues and Resolutions
+
+* https://issues.couchbase.com/browse/CBL-5723[CBL-5723 -- Fixed Replicator syncs from beginning when using prebuilt dbs synced from SG]
```

Comment:
"Fixed replicator syncs from beginning" seems awkward and unclear. After looking at the ticket, I'm thinking something like this might be clearer:

Ensured replicator syncs do not restart from the beginning when using prebuilt databases synced from Sync Gatweay.

But please check this with your SMEs.

Reply:
Done!